src/nvme/ioctl.c

```diff
@@ -344,14 +431,30 @@ int nvme_get_log_page(int fd, __u32 xfer_len, struct nvme_get_log_args *args)
 		args->len = xfer;
 		args->log = ptr;
 		args->rae = offset + xfer < data_len || retain;
+#ifdef CONFIG_LIBURING
+		if (n >= NVME_URING_ENTRIES) {
+			nvme_uring_cmd_wait_complete(&ring, n);
```

Comment:
I am fine with using uring but would it be possible to have it fallback during runtime to the ioctl when uring is not available, like on older kernels?